src/ckb/channel.rs

```diff
@@ -317,11 +324,13 @@ impl ChannelActor {
                     }),
                 };
                 if let RemoveTlcReason::RemoveTlcFulfill(fulfill) = command.reason {
-                    if tlc.payment_hash != blake2b_256(fulfill.payment_preimage).into() {
+                    let filled_payment_preimage: Hash256 =
```

Comment:
It would be better if we can rename `filled_payment_preimage` to `filled_payment_hash` as technically the result of a hash function is no longer a preimage.